portfolio/components/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+'use client'
+import Link from 'next/link'
+import Image from 'next/image';
+import { useState } from 'react'
+import { logo, sunIcon, moonIcon, hamburger, cross } from '../assets';
+import { useTheme } from 'next-themes';
+
+const MenuItems = ({ isMobile, active, setActive, setIsOpen }) => {
+    const navLinks = [
+        { url: '/', text: 'Home' },
+        { url: 'case-studies', text: 'Case Studies' },
+        { url: 'contact', text: 'Contact' },
+        { url: 'resume', text: 'Resume' },
+    ]
+
+    return (
+        <div className="flex flex-row justify-between list-none gap-4 items-center">
+            <ul className={`list-none flex ${isMobile
+                ? 'flex-col justify-end w-full border-b border-black'
+                : 'flex flex-row justify-between list-none gap-4'
+                }`}>
+                {navLinks.map((link, i) => (
```

Comment:
This is a simple suggestion for readability improvement. Inside your navLinks.map, you can destructure the URL and text properties right in the function parameters for clarity.
Again, what you have is fine, but to give you the very best feedback, I suggested this for readability. 

```
navLinks.map(({ url, text }) => (
  <Link href={url} key={url} ...
```